docs/pieces.md

```diff
@@ -202,21 +197,21 @@ In A2, methods were added in `construct`, which no longer exists. In A3, your cu
 
 #### Extending methods: generating example pieces
 
-Sometimes we want to extend an existing method we inherited from our base class, while still taking advantage of it. In A2 we did that "by hand," saving the old method in a variable. This was not intuitive for new developers.
+Sometimes we want to extend an existing method we inherited from our base class. In A2 we did that by saving the old method in a variable. This was not intuitive for new developers.
 
-A3 modules replace this with the `extendMethods` section, which provides a clear pattern. In the example above, the `generate` method is extended to generate a random `price`. Notice that `_super` is always the first argument here. Invoking it calls the old version of the method, as part of our implementation of the new one.
+A3 modules replace this process with the `extendMethods` section, providing a clear pattern. In the example above, the `generate` method is extended to generate a random `price`. Notice that `_super` is always the first argument. Invoking it calls the old version of the method, as part of our implementation of the new one.
 
 This allows us to run:
 
-```bash
+```sh
```

Comment:
This isn't consistent, but also not critical.

Reply:
Can we please go back to `bash`, we use bash-isms sometimes in our command lines. None of us use sh and none of us, to my knowledge, know what the forbidden "bash-isms" are (but `&&`, which we use like candy, is one of them).